frontend/src/lib/components/PropertyIcon.tsx

```diff
@@ -43,17 +51,15 @@ export const PROPERTIES_ICON_MAP = {
         ['Tablet']: <IconTablet />,
         ['Other']: <IconDevices />,
     },
-    $os: {
-        ['Mac OS X']: <IconMacOS />,
-        ['Windows']: <IconWindows />,
-        ['Linux']: <IconLinux />,
-        ['Android']: <IconAndroidOS />,
-        ['iOS']: <IconAppleIOS />,
-        ['Other']: <IconCogBox />,
-    },
+    $os: osIcons,
+    // some SDKs have $os_name instead of $os
```

Comment:
at least on Mobile, its always `$os_name`

Reply:
`$device_type` is always `Mobile`, `Desktop`, etc, not sure if it's useful here.
We could technically make it generic instead and just show a "mobile device" for Mobile, a computer icon for Desktop, etc.

Reply:
Yep, that's what happens... 👍